lstchain/data/lstchain_standard_config.json

```diff
@@ -73,7 +73,7 @@
     "max_depth": 30,
     "min_samples_leaf": 10,
     "n_jobs": -1,
-    "n_estimators": 150,
+    "n_estimators": 50,
```

Comment:
to be changed also in `lstchain_lhfit_config.json`?

Reply:
> to be changed also in `lstchain_lhfit_config.json`?

I guess so, but it is a question to @gabemery I guess

Reply:
I don't see a reason not to. Especially if there is no effect for the standard analysis before n_estimator = 5. I don't expect a simple different parametrization to significantly change the behaviour vs number of estimator.

@moralejo Can you share the code you used to remove estimators in already trained RFs? So that I can test it later?

Reply:
> I don't see a reason not to. Especially if there is no effect for the standard analysis before n_estimator = 5. I don't expect a simple different parametrization to significantly change the behaviour vs number of estimator.
> 
> @moralejo Can you share the code you used to remove estimators in already trained RFs? So that I can test it later?

Apologies, I had not read this yet. I will contact you @gabemery